### DISCUSSION TITLE
Magento 2 app Error during this cmd npx create-magento-app foldername

### DISCUSSION THREAD

**Discussion by Udeytechmagento2** [scandipwa/scandipwa#2383] [Install & Upgrade]
Unable to connect to MySQL, retrying... Attempt 2
› Unable to connect to MySQL, retrying... Attempt 3
› Unable to connect to MySQL, retrying... Attempt 4
› Unable to connect to MySQL, retrying... Attempt 5
› Unable to connect to MySQL, retrying... Attempt 6
› Unable to connect to MySQL, retrying... Attempt 7
› Unable to connect to MySQL, retrying... Attempt 8
› Unable to connect to MySQL, retrying... Attempt 9
› Unable to connect to MySQL, retrying... Attempt 10

**Comment by lianastaskevica** [comment]
Hello @Udeytechmagento2! Could you please provide more details on how to reproduce the issue?

**Comment by shivampmc** [comment]
I know it has been a bit late for me to answer.
You will get this error when you start your CMA for the first time. Re-start the CMS and this time it should run properly.

**Comment by alfredsgenkins** [comment]
@shivampmc What platform are you at? @ejnshtein have you seen this somewhere?

**Reply by shivampmc** [reply]
@alfredsgenkins  I am using Linux Mint 20 Cinnamon as of now.

**Reply by shivampmc** [reply]
Although I faced this in Manjaro KDE plasma. I did not try it in Linux Mint 20

**Reply by ejnshtein** [reply]
This is not an error, it just prints that it's trying to connect.
In the latest version, there's no such behaviour.

**Reply by shivampmc** [reply]
Not sure about the latest version but in the version I tried this was the case.

**Reply by ejnshtein** [reply]
I see.
Anyway, this messaging is fixed in the latest version which is currently available as alpha, even tho it doesn't breaks anything 🤔
Soon will be released in stable.

**Reply by shivampmc** [reply]
Is it with M2.4.2, you are talking about?

**Reply by ejnshtein** [reply]
Yeah
